astropy/tests/helper.py

```diff
@@ -465,48 +464,18 @@ def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
                                                                rtol, atol),
                                **kwargs)
 
-
+@deprecated("3.1",alternative='astropy.units.allclose')
 def quantity_allclose(a, b, rtol=1.e-5, atol=None, **kwargs):
     """
     Returns True if two arrays are element-wise equal within a tolerance.
 
     This is a :class:`~astropy.units.Quantity`-aware version of
     :func:`numpy.allclose`.
     """
-    import numpy as np
-    return np.allclose(*_unquantify_allclose_arguments(a, b, rtol, atol),
-                       **kwargs)
-
+    from ..units.quantity import allclose as quantity_allclose_units
+    quantity_allclose_units(a, b, rtol=1.e-5, atol=None, **kwargs)
 
+@deprecated("3.1",alternative='astropy.units._unquantify_allclose_arguments')
```

Comment:
This one doesn't have to be deprecated - it is a private function. However, you do need to import it inside `assert_quantity_allclose`

Reply:
Okay, but I do have to make a call to the new location of the function, in order to avoid code duplication.

Reply:
That why I noted that you have to import it in `assert_quantity_allclose`